### PR TITLE
docs(CONTRIBUTING): review

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

**Comment by vvo** [comment]
What we are missing in CONTRIBUTING is actually how to run tests on every API client. This is usually what the CONTRIBUTING.md contains: how to actually contribute, what's the repo organization. Basically what you would say to a fellow developer looking for contributing to your project code-wise.

WDYT?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,27 +13,33 @@ Note that some files are managed outside this repository and are committed autom
...
 ## Submitting a pull request
 
-Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
+Keep your changes as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
 
 Here are a few tips to increase the likelihood of being merged:
 
 - [ ] Write tests.
-- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) and follow (conventional commits)[https://conventionalcommits.org/].
```
Comment (Haroenv) [inline CONTRIBUTING.md]: markdown link format is backwards here 🙂

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,27 +13,33 @@ Note that some files are managed outside this repository and are committed autom
...
+If you want to make breaking changes, open an issue first to discuss it with maintainers.
 
 ## Label caption
 
 Labels across all repositories are normalized.
 
 | Label                                                                    | Meaning                                                                                |
 |--------------------------------------------------------------------------|----------------------------------------------------------------------------------------|
-| ![#050f2c](https://placehold.it/15/050f2c/000000?text=+) Do not merge    | PR should not be merged (decided by maintainers)                                       |
+| ![#050f2c](https://placehold.it/15/050f2c/000000?text=+) Do not merge    | PR should not be merged as it is (decided by maintainers)                                       |
```
Comment (Haroenv) [inline CONTRIBUTING.md]: s/as it is/as is/

**Review by Haroenv — COMMENTED** [review]
good reordering, two small typos/suggestions

**Comment by vvo** [comment]
About the missing "How to dev" part, we could definitely ask API maintainers to put that in a specific document and then link it from there.

File: CONTRIBUTING.md (hunk 1 above)
Comment (julienbourdeau) [reply CONTRIBUTING.md]: I'm personaly not a big fan of conventialcommits, but it seems like most engineers enjoy it here. I'm trying to organize my thoughts for the way to do changelogs in the API clients. I'll share them as soon as possible.

**Comment by julienbourdeau** [comment]
Thanks a lot @vvo and @Haroenv. I'll add the "how to dev" part 👌

**Pull request merged**